docs/resources/new_in_doc.md

```diff
@@ -6,6 +6,16 @@ description: Overview of major recent additions to Ibexa DXP documentation.
 
 This page contains recent highlights and notable changes in [[= product_name =]] documentation.
 
+## May 2024
+
+### PHP API
+
+- [PHP API Reference](https://doc.ibexa.co/en/master/api/php_api/php_api_reference/)
+
+### User management
```

Comment:
@juskora As recent activity was in the previous Users section, as this is more about users themselves and their rights, maybe it shoult be titled Users as well. What do you think?

```suggestion
### Users
```

Reply:
I think yes :)